src/main/kotlin/org/web3j/quorum/tx/QuorumTransactionManager.kt

```diff
@@ -54,12 +47,15 @@ class QuorumTransactionManager(
             val privateMessage = TransactionEncoder.signMessage(privateTransaction, credentials)
 
             signedMessage = setPrivate(privateMessage)
+            val hexValue = Numeric.toHexString(signedMessage)
```

Comment:
why repeat the two lines (hexValue and return) - when they are already there after the else clause

Reply:
We missed it :/